src/index.html

```diff
@@ -10,13 +24,95 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Moyo header</title>
+    <title>Mate academy</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
+    <header class="header">
+      <a
+        class="logo"
+        href="index.html"
+      >
+        <img
+          class="img"
+          src="images/logo.png"
+          alt="MOYO"
```

Comment:
Ensure the `alt` attribute for the logo image is descriptive enough for accessibility purposes. Consider using a more descriptive text than 'MOYO'.